tests/bootstrap_strings.py

```diff
@@ -1,4 +1,4 @@
-TEST_ENVIRONMENT_MANIFEST = '# The manifest for the "test" environment.\n# Read the full specification for the "Environment" type at:\n#  https://aws.github.io/copilot-cli/docs/manifest/environment/\n\n# Your environment name will be used in naming your resources like VPC, cluster, etc.\nname: test\ntype: Environment\n\n# Import your own VPC and subnets or configure how they should be created.\n# network:\n#   vpc:\n#     id:\n\n# Configure the load balancers in your environment, once created.\n\nhttp:\n  public:\n    certificates:\n      - ACM-ARN-FOR-test.landan.cloudapps.digital\n#   private:\n\n\n# Configure observability for your environment resources.\nobservability:\n  container_insights: true\n'
-PRODUCTION_ENVIRONMENT_MANIFEST = '# The manifest for the "production" environment.\n# Read the full specification for the "Environment" type at:\n#  https://aws.github.io/copilot-cli/docs/manifest/environment/\n\n# Your environment name will be used in naming your resources like VPC, cluster, etc.\nname: production\ntype: Environment\n\n# Import your own VPC and subnets or configure how they should be created.\n# network:\n#   vpc:\n#     id:\n\n# Configure the load balancers in your environment, once created.\n\nhttp:\n  public:\n    certificates:\n      - ACM-ARN-FOR-test.landan.cloudapps.digital\n#   private:\n\n\n# Configure observability for your environment resources.\nobservability:\n  container_insights: true\n'
+TEST_ENVIRONMENT_MANIFEST = "# The manifest for the \"test\" environment.\n# Read the full specification for the \"Environment\" type at:\n#  https://aws.github.io/copilot-cli/docs/manifest/environment/\n\n# Your environment name will be used in naming your resources like VPC, cluster, etc.\nname: test\ntype: Environment\n\n# Import your own VPC and subnets or configure how they should be created.\n# network:\n#   vpc:\n#     id:\n\nnetwork:\n  vpc:\n    cidr: '10.0.0.0/16'\n    subnets:\n      public:\n        - cidr: '10.0.0.0/24'\n          az: 'eu-west-2a'\n        - cidr: '10.0.1.0/24'\n          az: 'eu-west-2b'\n        - cidr: '10.0.2.0/24'\n          az: 'eu-west-2c'\n      private:\n        - cidr: '10.0.3.0/24'\n          az: 'eu-west-2a'\n        - cidr: '10.0.4.0/24'\n          az: 'eu-west-2b'\n        - cidr: '10.0.5.0/24'\n          az: 'eu-west-2c'\n\n# Configure the load balancers in your environment, once created.\n\nhttp:\n  public:\n    certificates:\n      - ACM-ARN-FOR-test.landan.cloudapps.digital\n#   private:\n\n\n# Configure observability for your environment resources.\nobservability:\n  container_insights: true\n"
```

Comment:
Maybe for a separate pull request, but could we do something like this to make these easier to grok and see what's actually changed?

```suggestion
TEST_ENVIRONMENT_MANIFEST = """# The manifest for the \"test\" environment.
# Read the full specification for the \"Environment\" type at:
#  https://aws.github.io/copilot-cli/docs/manifest/environment/

# Your environment name will be used in naming your resources like VPC, cluster, etc.
name: test
type: Environment

# Import your own VPC and subnets or configure how they should be created.
# network:
#   vpc:
#     id:
network:
  vpc:
    cidr: '10.0.0.0/16'
    subnets:
      public:
        - cidr: '10.0.0.0/24'
          az: 'eu-west-2a'
        - cidr: '10.0.1.0/24'
          az: 'eu-west-2b'
        - cidr: '10.0.2.0/24'
          az: 'eu-west-2c'
      private:
        - cidr: '10.0.3.0/24'
          az: 'eu-west-2a'
        - cidr: '10.0.4.0/24'
          az: 'eu-west-2b'
        - cidr: '10.0.5.0/24'
          az: 'eu-west-2c'

# Configure the load balancers in your environment, once created.

http:
  public:
    certificates:
      - ACM-ARN-FOR-test.landan.cloudapps.digital
#   private:


# Configure observability for your environment resources.
observability:
  container_insights: true
"""
```

Reply:
Sure - I will do it as part of this PR, as I would like to test the multi-AZ functionality anyway.

Reply:
Resolved in https://github.com/uktrade/copilot-tools/pull/102/commits/457fc8917c625c1a5d6a8e1bb89c401f077cd7a7.

Reply:
Great approach. Love it.